src/vcpkg/base/downloads.cpp

```diff
@@ -883,6 +883,43 @@ namespace vcpkg
         return true;
     }
 
+    bool azcopy_to_asset_cache(DiagnosticContext& context,
+                               StringView raw_url,
+                               const SanitizedUrl& sanitized_url,
+                               const Path& file)
+    {
+        auto azcopy_cmd = Command{"azcopy"};
```

Comment:
`azcopy` must be added to `vcpkg-tools.json` so we can automatically get a copy on the users behalf.

Reply:
This commit: https://github.com/microsoft/vcpkg/pull/45461/commits/c178e5494f9c9a4beaf76f28b09a5ec1b0df1b5f

Reply:
> `azcopy` must be added to `vcpkg-tools.json` so we can automatically get a copy on the users behalf.

PR for merge: https://github.com/microsoft/vcpkg/pull/45726